docs/testing/yaml_schema.md

```diff
@@ -0,0 +1,35 @@
+<!---
```

Comment:
Something seems off with the end result, when looking at preview and what got merged in github

https://github.com/project-chip/connectedhomeip/blob/master/docs/testing/yaml_schema.md